examples/simple/handlers.go

```diff
@@ -47,3 +48,14 @@ func registerHTTPers(service service.Host) []uhttp.RouteHandler {
 		uhttp.NewRouteHandler("/", handler),
 	}
 }
+
+func simpleFilter() uhttp.FilterFunc {
```

Comment:
why wrap this in a closure? in other words, why doesn't `simpleFilter` take ctx, w, r, next